.github/workflows/functional-test.yml

```diff
@@ -111,7 +111,7 @@ jobs:
     - uses: reactivecircus/android-emulator-runner@v2
       name: e2e_api${{ matrix.apiLevel }}
       with:
-        script: npx mocha --timeout 10m ./test/functional/driver-e2e-specs.js ./test/functional/commands/jetpack-compose-attributes-e2e-specs.js -g @skip-ci -i --exit
+        script: npx mocha --timeout 10m ./test/functional/driver-e2e-specs.js ./test/functional/commands/jetpack-compose-attributes-e2e-specs.js ./test/functional/webview/webatoms-e2e-specs.js -g @skip-ci -i --exit
```

Comment:
I'd like to enable other func test later, so this is to enable this test only for this pr